gatorgrade/main.py

```diff
@@ -16,6 +19,9 @@ def gatorgrade(
     """Run the GatorGrader checks in the gatorgrade.yml file."""
     # check if ctx.subcommand is none
     if ctx.invoked_subcommand is None:
+        checks = parse_config(FILE)
+        checks = run_commands_and_return_results(FILE)
```

Comment:
In PR #68, it appears that the main function for the Output team is `run_and_display_command_checks`, whose parameter is not a file path. So, this function call should be updated to use the correct function and correct argument value.